cypress/e2e/pages/articles/testsForAMPOnly.js

```diff
@@ -1,28 +1,33 @@
 /* eslint-disable import/prefer-default-export */
-import appConfig from '../../../../src/server/utilities/serviceConfigs';
-import { getBlockData, getVideoEmbedUrl } from './helpers';
 import { ampOnly as mostReadAssertions } from '../mostReadPage/mostReadAssertions';
 
 // TODO: Remove after https://github.com/bbc/simorgh/issues/2959
 const serviceHasFigure = service =>
   ['arabic', 'news', 'pashto', 'persian', 'urdu'].includes(service);
 
+const articleHasPlayer = articleId =>
+  [
+    'cgwk9w4zlg8o', // pidgin/articles/cgwk9w4zlg8o on LIVE
+    'cj7xrxz0e8zo', // news/articles/cj7xrxz0e8zo on LIVE
+    'c25rp5glj5qo', // persian/articles/c25rp5glj5qo on LIVE
+    'cwl08rd38l6o', // pidgin/articles/cwl08rd38l6o on TEST
+    'cej3lzd5e0go', // persian/articles/cej3lzd5e0go on TEST
+  ].includes(articleId);
+
 // For testing features that may differ across services but share a common logic e.g. translated strings.
 export const testsThatFollowSmokeTestConfigForAMPOnly = ({
   service,
   pageType,
   variant,
 }) => {
-  let articlesData;
   describe(`Running testsForAMPOnly for ${service} ${pageType}`, () => {
+    let articleId;
     before(() => {
-      cy.getPageData({ service, pageType: 'article', variant }).then(
-        ({ body }) => {
-          articlesData = body;
-        },
-      );
+      cy.url().then(url => {
+        // eslint-disable-next-line prefer-destructuring
+        articleId = url.match(/\/([^/]+?)(?:\.[^/.]+)?$/)[1];
```

Comment:
Let's use destructuring

```suggestion
        const [, articleId] = url.match(/\/([^/]+?)(?:\.[^/.]+)?$/);
```